facade/src/main/java/org/jboss/pnc/facade/providers/ProductMilestoneProviderImpl.java

```diff
@@ -489,6 +492,27 @@ private String parseArtifactTypeFromDeployPath(String deployPath, RepositoryType
         return "";
     }
 
+    private String parseArtifactClassifierFromDeployPath(String deployPath, RepositoryType repositoryType) {
+        switch (repositoryType) {
+            case MAVEN: {
+                var mavenCoordinates = ArtifactCoordinatesUtils.parseMavenCoordinates(deployPath);
+
+                if (mavenCoordinates != null) {
+                    String classifier = mavenCoordinates.getClassifier();
+                    return classifier != null ? classifier : "";
```

Comment:
Why not keeping the classifier null? It just clutters the JSON response with empty strings. Similar thing with NPM artifacts, they do not have a concept of classifier.